src/js/recommender.js

```
@@ -17,17 +17,23 @@ function getRecommendations(resumeSkills = [], githubProfile = null) {
-  if (githubProfile) {
-    (githubProfile.languages || []).forEach(l => userLanguages.add(normalize(l.toLowerCase())));
-    (githubProfile.topics || []).forEach(t => userTopics.add(normalize(t.toLowerCase())));
-  }
-
-  resumeSkills.forEach(s => {
-    const skill = normalize(s.toLowerCase());
-    userLanguages.add(skill);
-    userTopics.add(skill);
+  const userTopics = [
+  ...new Set([
+    ...(githubTopics || []).map(t => t.toLowerCase()),
```

ESLint

[error] 'githubTopics' is not defined.

(no-undef)